### PR TITLE
feat: add header

### PR DESCRIPTION
**Pull request by zalevskyi**
Please review header
[Task details](https://trello.com/c/xAnbODhb/15-implement-header)
Overall it seems that task lacks sidebar toggle button

### CODE REVIEW

File: frontend/src/components/header/components/nav-profile.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <span className="text-dark">{userName}</span>
+      </Dropdown.Toggle>
+
+      <Dropdown.Menu>
+        <Dropdown.Item href="#/profile">
+          <img src={profileIcon} alt="profile icon" />
+          <span>Profile</span>
+        </Dropdown.Item>
+        <Dropdown.Divider />
+        <Dropdown.Item href="#/workspace">Select Workspace</Dropdown.Item>
```
Comment (aleksandr-kuharenko) [inline frontend/src/components/header/components/nav-profile.tsx:26]: Use "Link" component and Enum for routes. For all other links as well.
```suggestion
        <Dropdown.Item><Link to={AppRoute.WORKSPACES}>Select Workspace</Link></Dropdown.Item>
```

File: frontend/src/components/header/header.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import Navbar from 'react-bootstrap/Navbar';
+import NavProfile from './components/nav-profile';
+import NavNotification from './components/nav-notification';
+import Search from './components/search';
+import './header.css';
```
Comment (aleksandr-kuharenko) [inline frontend/src/components/header/header.tsx:5]: We use Scss for styles, please change the extension

File: frontend/src/assets/img/notification-none.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24" fill="none" stroke="#6c757d" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-bell-off align-middle"><path d="M13.73 21a2 2 0 0 1-3.46 0"></path><path d="M18.63 13A17.89 17.89 0 0 1 18 8"></path><path d="M6.26 6.26A5.86 5.86 0 0 0 6 8c0 7-3 9-3 9h14"></path><path d="M18 8a6 6 0 0 0-9.33-5"></path><line x1="1" y1="1" x2="23" y2="23"></line></svg>
```
Comment (aleksandr-kuharenko) [inline frontend/src/assets/img/notification-none.svg:1]: Lets use bootstrap [icons](https://icons.getbootstrap.com/#styling). 
For example: `<i class="bi bi-person"></i>`

Here is the list of icons for the header:
notification - https://icons.getbootstrap.com/icons/bell-slash/
profile icon - https://icons.getbootstrap.com/icons/person/
search icon - https://icons.getbootstrap.com/icons/search/
avatar mock - https://icons.getbootstrap.com/icons/person-circle/

**Review by aleksandr-kuharenko — CHANGES_REQUESTED** [review]

**Comment by aleksandr-kuharenko** [comment]
> Overall it seems that task lacks sidebar toggle button

Let's skip it for now. We will add this in the next iterations.

**Review by aleksandr-kuharenko — APPROVED** [review]

**Review by ursarik — APPROVED** [review]

**Pull request merged**